### PR TITLE
chore: yaml issue template improvements

### PR DESCRIPTION
**Pull request by UrAvgDeveloper**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
         Thank you for reporting the issue you are facing.
 
         Please complete the below form to ensure we have the necessary details to assist you.
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please answer the following questions for yourself before submitting an issue
+      options:
+        - label: I am running the latest version
```
Comment (bryanchriswhite) [inline .github/ISSUE_TEMPLATE/bug-report.yml:18]:
```suggestion
        - label: I am running the latest version  [![cosmpy PyPI version](https://img.shields.io/pypi/v/cosmpy?label=cosmpy%20%28PyPI%29)]
```

**Comment by bryanchriswhite** [comment]
The shield is still missing next to the version prompt.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -1,59 +1,66 @@
...
 title: "Bug report: <short description>"
-labels: ["bug: product"]
+labels: ["bug", "unconfirmed"]
 body:
   - type: markdown
     attributes:
       value: |
         Thank you for reporting the issue you are facing.
 
         Please complete the below form to ensure we have the necessary details to assist you.
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:11]:
```suggestion
        Please complete the following so we can have the details necessary to assist you.
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -1,59 +1,66 @@
...
+    id: prerequisites
     attributes:
-      label: Description
-      description: Provide a short description of the issue you are facing
-    validations:
-      required: true
+      label: Prerequisites
+      description: Please read and confirm before submitting an issue.
+      options:
+        - label: I am running the latest CosmPy version.
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:18]: Can you integrate the version here using `cosmpy pypi version`? 

Something like:

```suggestion
        - label: I am running the latest CosmPy version ![cosmpy pypi version](https://img.shields.io/pypi/v/cosmpy).
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -1,59 +1,66 @@
...
+          required: true
+        - label: I checked the [documentation](http://docs.fetch.ai/cosmPy/) and found no answer to my problem.
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/bug-report.yml:20]:
```suggestion
        - label: I checked the [documentation](http://docs.fetch.ai/CosmPy/) and found no answer to my problem.
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 5]
```diff
@@ -1,41 +1,51 @@
...
         Thank you for suggesting an idea to make better.
 
         Please complete the below form to ensure we have all the details to get things started.
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please read and confirm before submitting a feature request.
+      options:
+        - label: I checked the [documentation](http://docs.fetch.ai/cosmPy/) and made sure this feature does not already exist.
```
Comment (5A11) [inline .github/ISSUE_TEMPLATE/feature-request.yml:18]:
```suggestion
        - label: I checked the [documentation](http://docs.fetch.ai/CosmPy/) and made sure this feature does not already exist.
```

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 3 above)
Comment (bryanchriswhite) [reply .github/ISSUE_TEMPLATE/bug-report.yml:18]: I would suggest making it a link to the PyPI page
```suggestion
        - label: I am running the latest CosmPy version [![cosmpy PyPI version](https://img.shields.io/pypi/v/cosmpy?label=cosmpy%20%28PyPI%29)].
```

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**